cdap-ui/app/features/workflows/controllers/tabs/runs/status-ctrl.js

```diff
@@ -0,0 +1,192 @@
+angular.module(PKG.name + '.feature.workflows')
+  .controller('WorkflowsDetailRunStatusController', function($state, $scope, MyDataSource, amMoment, $filter) {
+    var dataSrc = new MyDataSource($scope),
+        filterFilter = $filter('filter'),
+        basePath = '/apps/' + $state.params.appId + '/workflows/' + $state.params.programId;
+    $scope.moment = amMoment;
+    $scope.moment.changeLocale('en');
```

Comment:
I'm assuming that would change the locale for all amMoment instances throughout the app. What is it for, why is it here?